idd/Energy+.idd.in

```diff
@@ -91500,10 +90780,11 @@ Table:MultiVariableLookup,
        \min-fields 27
   A1 , \field Name
        \required-field
-       \reference AllCurves
-       \reference UniVariateTables
-       \reference BiVariateTables
-       \reference MultiVariateTables
```

Comment:
May need to keep this reference for objects that only work with multivariate tables?  I guess there aren't really any curve types that are tagged as MultiDimCurves?

Reply:
Additional possible terms to throw around to replace curves or tables:

- Formulation
- Map
- Relationship

But i do like Function you mentioned above @nealkruis.

Also, if we want to really be pedantic, the curves here aren't one dimensional, two dimensional, etc.  That is the dimension of the independent variables.  The curve would be that plus 1.  So OneDimCurves would be OneDimIndependentVariables.  I kinda like the Univariate and Bivariate terminology as it describes that the function is allowed to vary in that number of independent variable dimensions.  

Maybe `\reference UnivariateFunctions`, `\reference BivariateFunctions` ?

Reply:
`\reference TriVariateFunctions`? `\reference QuadVariateFunctions`? If we're okay with those strange sounding terms, I'll make the change.

Reply:
Weird doesn't mean wrong, so I'm OK with it, but I would love another vote or two from others.  There's still a good chance there's better terminology out there.  Where's the academics out there?  @mitchute @jasondegraw ?

Reply:
Function is fine with me.

Reply:
Uni/Bi/Tri/Quad/Multi-variate all look OK to me.

Also, if we're curving fitting shouldn't we level-up with a generalized NN model? 😉

Reply:
OK, seems like we have enough support for *VariateFunctions.  Make it so, @nealkruis!